docker/base/centos7/Dockerfile

```diff
@@ -28,10 +28,11 @@ RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
     vim \
     wget \
     which \
-    yum-utils \
- && sed -i 's/http:\/\/vault.centos.org/https:\/\/vault.centos.org/g' /etc/yum.repos.d/* \
- && yum upgrade ca-certificates
+    yum-utils
 
+RUN sed -i 's/http:\/\/vault.centos.org/https:\/\/vault.centos.org/g' /etc/yum.repos.d/* \
```

Comment:
Can you explain what's going on here? It doesn't look like a change, just a move from one RUN to another.

Reply:
Yeah, I just moved it to another RUN command. It seemed to make sense to me that this should have its own since it was, for a few days, resulting in errors. I am not sure why, but now it is required for the build to work. Perhaps a future update to the centos7 base image might resolve the issue this addresses and we can remove it.